### PR TITLE
Implementation for module

### PR DESCRIPTION
**Pull request by kaka89**

### CODE REVIEW

File: src/main/java/io/reactivesw/admin/authentication/application/controller/ModuleController.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+   * @param id id
+   * @return
+   */
+  @GetMapping(Router.MODULE_ROOT_WITH_ID)
+  public ModuleView getModuleById(@PathVariable String id) {
+    LOG.info("Enter. id: {}.", id);
+
+    ModuleView view = moduleApplication.getById(id);
+
+    LOG.info("Exit. view: {}.", view);
```
Comment (davis999) [inline src/main/java/io/reactivesw/admin/authentication/application/controller/ModuleController.java]: should be moduleView.

File: src/main/java/io/reactivesw/admin/authentication/application/service/ModuleApplication.java [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  public List<ModuleView> getAll() {
+    LOG.debug("Enter.");
+
+    List<Module> moduleList = moduleService.getAll();
+    LOG.trace("model list: module: {}.", moduleList);
+
+    List<ModuleView> viewList = ModuleMapper.toView(moduleList);
+
+    LOG.debug("Exit. moduleView size: {}.", viewList.size());
+    LOG.trace("View list: moduleView: {}.", viewList);
```
Comment (davis999) [inline src/main/java/io/reactivesw/admin/authentication/application/service/ModuleApplication.java]: why log twice in trace?
that are similar.

File: src/main/java/io/reactivesw/admin/authentication/domain/service/ModuleService.java [hunk 3]
```diff
@@ -40,4 +44,37 @@ public Module create(Module module) {
...
+   *
+   * @return List of modules
+   */
+  public List<Module> getAll() {
+    LOG.debug("Enter.");
+
+    List<Module> moduleList = this.moduleRepository.findAll();
+
+    LOG.debug("Exit. moduleSize: {}.", moduleList.size());
+    LOG.trace("Exit. module: {}.", moduleList);
```
Comment (davis999) [inline src/main/java/io/reactivesw/admin/authentication/domain/service/ModuleService.java]: i think trace log do not need `Exit` tag.

File: src/main/java/io/reactivesw/admin/authentication/infrastructure/update/UpdateActionUtils.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+  /**
+   * Update module.
+   */
+  public static final String UPDATE_MODULE = "updateModule";
+
+  /**
+   * Default private constructor.
+   */
+  private UpdateActionUtils() {
```
Comment (davis999) [inline src/main/java/io/reactivesw/admin/authentication/infrastructure/update/UpdateActionUtils.java]: do not need this constructor.

File: src/main/java/io/reactivesw/admin/authentication/domain/service/ModuleService.java (hunk 3 above)
Comment (kaka89) [reply src/main/java/io/reactivesw/admin/authentication/domain/service/ModuleService.java]: done.

File: src/main/java/io/reactivesw/admin/authentication/infrastructure/update/UpdateActionUtils.java (hunk 4 above)
Comment (kaka89) [reply src/main/java/io/reactivesw/admin/authentication/infrastructure/update/UpdateActionUtils.java]: done.

File: src/main/java/io/reactivesw/admin/authentication/application/controller/ModuleController.java (hunk 1 above)
Comment (kaka89) [reply src/main/java/io/reactivesw/admin/authentication/application/controller/ModuleController.java]: done.

File: src/main/java/io/reactivesw/admin/authentication/infrastructure/update/UpdateActionUtils.java (hunk 4 above)
Comment (kaka89) [reply src/main/java/io/reactivesw/admin/authentication/infrastructure/update/UpdateActionUtils.java]: done.

**Pull request merged**